src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -419,6 +419,31 @@ After configuring the OAuth endpoints (with Cognito Hosted UI), you can integrat
 
 </InlineFilter>
 
+```ts
+import { Hub } from "aws-amplify/utils";
+import { signInWithRedirect, getCurrentUser } from "aws-amplify/auth";
+
+Hub.listen("auth", async ({ payload }) => {
+  switch (payload.event) {
+    case "signInWithRedirect":
```

Comment:
nit: should show the `signedIn` event as the example here? This event comes with the result of `getCurrentUser()` in the payload. Wdyt?

Reply:
wanted just expand the current examples that we have on the docs